packages/zod-validator/README.md

```diff
@@ -7,7 +7,7 @@ You can write a schema with Zod and validate the incoming values.
 
 ```ts
 import { z } from 'zod'
-import { zValidator } from '../src'
+import { zValidator } from '@hono/zod-validator'
```

Comment:
Unrelated change for this PR so I can remove if necessary.

Consumers of the `@hono/zod-validator` library would be importing it directly.